docs/_includes/uri-include.adoc

```diff
@@ -4,24 +4,41 @@
 This content is included in the user manual
 ////
 
-The include directive can insert content directly from a URI.
+The include directive can insert content directly from a URI when the `allow-uri-read` parameter is:
 
-Here's an example that demonstrates how to include AsciiDoc content:
+* passed on the command-line,
+* set as a parameter inside `asciidoctor.js` implementations.
```

Comment:
Instead of "set as a parameter inside `asciidoctor.js` implementations", this should read as:

Passed as an attribute to the Asciidoctor API (Asciidoctor, AsciidoctorJ or Asciidoctor.js).